api/src/main/java/org/openmrs/module/ugandaemrreports/reports/SetUpRecencyHTSClientCardDataExportReport2019.java

```diff
@@ -174,6 +174,8 @@ private DataSetDefinition dataSetDefinition() {
         dsd.addColumn("referred_location", sdd.definition("referralPlace", getConcept("dce015bb-30ab-102d-86b0-7a5022ba4115")), "onOrAfter=${startDate},onOrBefore=${endDate}", new ObsDataConverter());
         dsd.addColumn("dhis2_uuid", sdd.getDHIS2Uuid(), (String) null);
         dsd.addColumn( "encounter_uuid", df.getHTSEncounterUuid(), (String) null);
+        dsd.addColumn("recency viral load result", sdd.definition("recencyVLresult", getConcept("dc8d83e3-30ab-102d-86b0-7a5022ba4115")), "onOrAfter=${startDate},onOrBefore=${endDate}", new ObsDataConverter());
+        dsd.addColumn("recency viral load undetectable", sdd.definition("recencyVLundetectable", getConcept("dca1269d-30ab-102d-86b0-7a5022ba4115")), "onOrAfter=${startDate},onOrBefore=${endDate}", new ObsDataConverter());
```

Comment:
@Mwanje Change `recency viral load undetectable` to `is_recency_vl_undetectable`